webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PolicyWriterService.scala

```diff
@@ -613,7 +614,23 @@ class PolicyWriterServiceImpl(
     res
   }
 
-  // just write an empty file for now
+
+  private[this] def writeDirectiveCsv(paths: NodePromisesPaths, policies: Seq[Policy], policyMode : GlobalPolicyMode) =  {
+    val path = new File(paths.newFolder, "rudder-directives.csv")
+
+    val csvContent = for {
+      policy <- policies
+    } yield {
+      s"""${policy.id.directiveId.value};${policy.policyMode.getOrElse(policyMode.mode).name};${policy.technique.generationMode.name};${policy.technique.agentConfig.runHooks.nonEmpty};${policy.technique.id.name};${policy.technique.id.version};"${policy.directiveOrder.value}""""
```

Comment:
So, we converged with a normalized solution and still easy to parse in shell!
- fully quoted csv, ie: `"field 1","field 2","some \" in field 3","some , in last field"
- the first fields MUST be comma-free, only the last can contain comma. 
In shell, we will use `cut` with the numbered fields for first columns (uuid,etc) and a `cut` "everything remaining" for last part.

Reply:
The first fields are missing the quotes